_slides/17-vmanuppelli.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to VINCE's slide deck!"
```

Comment:
Not clear at this time who own this slidedeck: you, or us?

Reply:
@vmanuppelli testing!